app/views/order_items/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Your Cart</h1>
+
+  <ul>
+    <% total = 0 %>
+    <% @order_items.each do |item| %>
+      <% product = Product.find(item.product_id) %>
+      <li>
+        <%= product.name %>
+        <ul>
+          <li>Price: <%= product.price %></li>
+          <li>Quantity: <%= item.quantity %>
+            <% if product.stock > 1 %>
+              <%= form_for(item, url: update_cart_path(item.id), method: :patch) do |f| %>
+                <%= f.label "Change Quantity" %>
+                <%= f.select(:quantity, options_for_select((1..product.stock)), include_blank: true) %>
```

Comment:
I really like how you had the range for `options_for_select` dependent on the stock of the product.